jenkins/scripts/select-compiler.sh

```diff
@@ -105,6 +105,22 @@ elif [ "$SELECT_ARCH" = "AIXPPC" ]; then
       ;;
   esac
 
+  case $NODE_NAME in
```

Comment:
don't need a new `case...`, add the block to the existing one at line 91, and this `*aix71* )` block would get inserted after line 105 (the `;;`)